### DISCUSSION TITLE
Another working device. Baseus Orange Dot AI Wireless Presenter, abc8:ca08

### DISCUSSION THREAD

**Discussion by SirLesP** [jahnf/Projecteur#217] [General]
When Projecteur is started with -D abc8:ca08 this device functions 'out of the box' with Projecteur. The button labelled with a mouse symbol controls the on screen pointer of Projecteur and the forward and back buttons also work. (The red button is a physical laser, but doesn't appear mappable to the virtual laser.) Some of the side buttons are not set up correctly, but this can of course be tweaked in the Projecteur key mappings. This particular device is readily available on (say) AliExpress but doesn't seem to feature on European sites. It seems well made.
![Baseus](https://github.com/jahnf/Projecteur/assets/146490207/7385a888-27e2-4414-9e0e-ac14628dcda1)
So thanks for a very useful program!

**Comment by jahnf** [comment]
Thank you for the details. 👍🏻

**Comment by SirLesP** [comment]
Additional detail for the Baseus; it reports as "iTing BaseusPresent (abc8:ca08)".
Also note that the Baseus devices currently readily obtainable in Europe are laser only, not 'air mouse'.
